lib/system/sysio.nim

```diff
@@ -37,6 +37,8 @@ proc fread(buf: pointer, size, n: int, f: File): int {.
 proc fseek(f: File, offset: clong, whence: int): int {.
   importc: "fseek", header: "<stdio.h>", tags: [].}
 proc ftell(f: File): int {.importc: "ftell", header: "<stdio.h>", tags: [].}
+proc ferror(f: File): int {.importc: "ferror", header: "<stdio.h>", tags: [].}
+proc feof(f: File): int {.importc: "feof", header: "<stdio.h>", tags: [].}
```

Comment:
The static data is no longer sent to the clients (.js, .css etc.). I did not spent time to investigate after seeing this comet: ``` # do not blame me; blame the ANSI C standard this is so brain-damaged ``` inside the endOfFile proc.